lib/message-routing/message-router.js

```diff
@@ -27,7 +27,7 @@ class MessageRouter {
     const messageContent = newMessage.message;
     getReporter().incrementCounter(METRIC_NAMES.MESSAGE_RATE, 1);
     // No api to get roles, we gotta STORE IT
-    this.messageRelay.relayMessageToListeners(newMessage);
+    this.messageRelay.relayMessageToListeners('message', newMessage);
```

Comment:
I get confused between messages and the more specific `MSG`-type messages sometimes. Maybe emitting `msg` instead of `message` for those types of messages would help.